README.md

```diff
@@ -36,7 +36,7 @@ written in the host language which starts a Prybar-compatible REPL.
 
 | language                  | eval | eval expression | eval file | repl | repl like eval | set prompt |
 | ------------------------- | ---- | --------------- | --------- | ---- | -------------- | ---------- |
-| Clojure                   | ✔    | ✔               | ✔         | ✔    | ✔              | -          |
+| Clojure                   | ✔    | ✔               | ✔         | ✔    | ✔              | ✗          |
```

Comment:
I absolutely hear your point about a static `PS1` causing confusion over namespaces, and standalone uses of `prybar-clojure` don't have to support it. However, being able to set the prompt is a key part of repl.it's branding, so we need to retain the ability to set `PS1`. Does Clojure offer a nice way to have one part of the prompt be static and other parts be dynamic?

Reply:
I actually like your idea about this! 🙂 As it turns out, it's very achievable thanks to the flexibility in customization provided by the Clojure REPL.[1] In fact, I should've done it the way you suggested in the first place. I made some changes to `languages/clojure/main.go` so that we can keep both the REPL.it branding and the current namespace indicator in the prompt. Here's a preview of what It looks like after the said changes:
```sh
$ prybar-clojure -q -i -ps1 '👉 ' main.clj
```
```clojure
hello, main
user 👉 (clojure-version)
"1.10.1"
user 👉 a
Syntax error compiling at (/tmp/prybar_clojure_repl.clj772020608:9:1).
Unable to resolve symbol: a in this context
user 👉 main/a
42
user 👉 (in-ns 'main)
#object[clojure.lang.Namespace 0x29182679 "main"]
main 👉 a
42
main 👉 
```
Let me know your thoughts. 🙂 I'll push all the changes as soon as I'm done fixing the tests.

[1] https://insideclojure.org/2020/02/11/custom-repl/